bin/celeborn-ratis

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function printUsage {
+  echo "Usage: Celeborn ratis COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
```

Comment:
maybe `Celeborn ratis` => `sbin/celeborn-ratis`?

Reply:
A little weird, when user got the Usage message, user must know it's under bin

Reply:
Then let's use the command itself `Celeborn ratis` => `celeborn-ratis`

Reply:
> Then let's use the command itself `Celeborn ratis` => `celeborn-ratis`

Yea, better